src/constants/generic.js

```diff
@@ -0,0 +1,4 @@
+export const FUNCTION = 'function';
+export const STRING = 'string';
+export const CHILDREN = 'children';
+export const COLOR = 'color';
```

Comment:
This doesn't seem to be used? We don't need generics such as these if they're well defined.

Reply:
oh! I forgot to remove that file on yesterday’s commit. Will update.